tools/class-redirector/class-redirector/ClassRedirectorTask.cs

```diff
@@ -0,0 +1,95 @@
+using System;
```

Comment:
MSBuild tasks should go in the msbuild/Xamarin.MacDev.Tasks/Tasks directory, together with all our other tasks.